### PR TITLE
Add GraalVM for JDK 21 Community 21.0.1

### PR DESCRIPTION
**Pull request by manofthepeace**
Not sure I did it properly. Apologies if I did not.

### CODE REVIEW

File: quarkus-native-s2i/graalvm.yaml [hunk 1]
```diff
@@ -17,6 +17,15 @@ images:
       - arch: arm64
         sha: bb4e92cf7eae91e474061aeae5ae75053a65cd558dbee76947827bf54d1b30a5
 
+  # https://github.com/graalvm/graalvm-ce-builds/releases/tag/jdk-21.0.1
+  - java-version: 21.0.1
+    tags: jdk-21
```
Comment (zakkak) [inline quarkus-native-s2i/graalvm.yaml:22]: You need to remove `tags: jdk-21` from 21.0.0

File: quarkus-graalvm-builder-image/graalvm.yaml [hunk 2]
```diff
@@ -25,3 +25,12 @@ images:
         sha: 6c422941ccc58be5b891bb6499feeb72cd2b74d6729a29bf1fb8cc1a7d58b319
       - arch: arm64
         sha: bb4e92cf7eae91e474061aeae5ae75053a65cd558dbee76947827bf54d1b30a5
+
+  # https://github.com/graalvm/graalvm-ce-builds/releases/tag/jdk-21.0.1
+  - java-version: 21.0.1
+    tags: jdk-21
```
Comment (zakkak) [inline quarkus-graalvm-builder-image/graalvm.yaml:31]: You need to remove `tags: jdk-21` from 21.0.0

**Review by zakkak — CHANGES_REQUESTED** [review]
Hi @manofthepeace, thanks for the PR. Please have a look at my comments.

**Comment by manofthepeace** [comment]
Thank you @zakkak . Done

**Review by zakkak — APPROVED** [review]
LGTM. Thanks @manofthepeace

**Comment by cescoffier** [comment]
Thanks!

**Comment by manofthepeace** [comment]
Can this be pushed to quay?

**Comment by cescoffier** [comment]
I just triggered the publication.

**Pull request merged**